src/index.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import createReactClass from 'create-react-class';
 import { render } from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
-import 'file-loader?name=index.html!../example/index.html';
```

Comment:
I think this should be a separate PR, since it doesn't deal directly with this.

Reply:
ok, see #701